draft-ietf-rats-architecture.md

```diff
@@ -1013,6 +1013,46 @@ an Attestation Result for any other purpose.
 
 # Security Considerations
 
+## Attester and Attestation Key Protection
+
+(this is a crude first pass to see what interest there is in this)
```

Comment:
Yes I think this content is useful, so comments below try to clean it up a bit.  Also some parts below might be more generalizable beyond just the manufacturer, as noted below.